src/main/kotlin/com/team4099/robot2023/subsystems/manipulator/Manipulator.kt

```diff
@@ -50,27 +50,20 @@ class Manipulator(val io: ManipulatorIO) : SubsystemBase() {
 
   // checks if motor current draw is greater than given threshold and if rollers are intaking
   // last condition prevnts current spikes caused by starting to run intake from triggering this
-  val intakingGamePiece: Boolean
+  val hasCube: Boolean
     get() {
-      return inputs.rollerStatorCurrent >= ManipulatorConstants.INTAKE_CURRENT_THRESHOLD &&
-        (
-          rollerState == ManipulatorConstants.RollerStates.CUBE_IN ||
-            rollerState == ManipulatorConstants.RollerStates.CONE_IN
-          ) &&
+      return inputs.rollerStatorCurrent >= ManipulatorConstants.CUBE_CURRENT_THRESHOLD &&
+        rollerState == ManipulatorConstants.RollerStates.CUBE_IN &&
```

Comment:
What happens after you release the "intake" button, and the roller state is no longer `CUBE_IN`?